platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ExportMigrationManagerImpl.java

```diff
@@ -217,4 +239,15 @@ public Path getExportFile() {
   Map<String, Object> getMetadata() {
     return metadata;
   }
+
+  private String getPropertiesFile(Path path) {
+    return AccessUtils.doPrivileged(
+        () -> {
+          try {
+            return new String(Files.readAllBytes(path));
```

Comment:
❗️ You have to account for encoding when reading the bytes into a String. i.e. pass `Charsets.UTF_8` into the String constructor as the second parameter.